README.md

```diff
@@ -1,2 +1,58 @@
 # publicly-available-microscopy-data
-Compiling and describing publicly available microscopy datasets
+## Purpose
+Compile and describe publicly available microscopy datasets using alpha and beta diversity measures. We extract metadata for each well and compute diveristy statistics for image attributes including but not limited to `screen id`, `genotype`, `phenotype`, `imaging method`, `cell line`, and `organism`. These measures give us insight into the quality of these datasets for use in image-based profiling methods.
```

Comment:
```suggestion
Compile and describe publicly available microscopy datasets using alpha and beta diversity measures.
We extract metadata for each well and compute diversity statistics for image attributes including but not limited to `screen id`, `genotype`, `phenotype`, `imaging method`, `cell line`, and `organism`.
These measures give us insight into the quality of these datasets for use in image-based profiling methods.
```

In markdown files, we use one-sentence-per-line https://news.ycombinator.com/item?id=31808093

Reply:
Sentences in purpose section were changed to one sentence per line.

Added in 72106573619716dbd5eaa7d5aea1375ee08e10e8.